tests/e2e/scale/noobaa/test_scale_namespace_rpc.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    MCGTest,
+    skipif_ocs_version,
+    vsphere_platform_required,
+    scale,
+)
+from ocs_ci.ocs import constants
+
+logger = logging.getLogger(__name__)
+
+
+@scale
+class TestScaleNamespace(MCGTest):
+    """
+    Test creation of a namespace scale resource
+    """
+
+    @skipif_ocs_version("<4.6")
+    @pytest.mark.polarion_id("OCS-2516")
+    def test_scale_namespace_bucket_creation(self, ns_resource_factory, bucket_factory):
```

Comment:
We should skip this test for vmware/BM right ? since there won't be aws-s3 bucket.